pkg/openstack/loadbalancer.go

```diff
@@ -663,6 +697,7 @@ func (lbaas *LbaasV2) ensureFloatingIP(ctx context.Context, clusterName string,
 				if err != nil {
 					return "", err
 				}
+				_ = lbaas.updateFloatingIPTag(ctx, floatIP, svcConf.lbName, true)
```

Comment:
I wonder why do you skip `err`?

Reply:
If you think this and the `add` below need to return err, tell me and I will add it.